plugin.go

```diff
@@ -282,3 +354,17 @@ func (p *Plugin) walkFiles() ([]string, error) {
 
 	return items, err
 }
+
+func extractDir(destination string) string {
```

Comment:
this is just a line function, you can remove it and use `filepath.Dir(destination)` directly

Reply:
Removed the function and used it directly.